addon/components/frost-select-dropdown.js

```diff
@@ -91,7 +91,7 @@ export default Component.extend({
     ]
       .join(';')
 
-    return Ember.String.htmlSafe(style)
+    return EmberString.htmlSafe(style)
```

Comment:
String is a Javascript class that is globally available. If we import as String we override that which could have unforeseen side effects.